app/Libraries/OsuAuthorize.php

```diff
@@ -32,6 +32,7 @@
 class OsuAuthorize
 {
     const REQUEST_ATTRIBUTE_KEY = 'auth_map';
+    const REQUEST_LIO_KEY = 'is_lio_request';
```

Comment:
it hasn't been "legacy" anymore and the newer endpoints are in `InterOp` (also the old controller should eventually be split and moved there)

`lio` name should only remain in the url and nowhere else (or maybe even add a copy route and eventually rename everything).